tigerpath/majors_and_certificates/majors/FIT_2018.json

```diff
@@ -0,0 +1,362 @@
+                  "double_counting_allowed": true,
+                  "course_list": [
+                    "ITA 207I",
+                    "ITA 208"
+                  ]
+                },
+                {
+                  "name": "200-Level Course",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "FIT recommends that students complete at least one more 200-level course (such as FRE 211, 215, 217, 221, 222, 224, 225, 230, or ITA 208, 209, 220, 225) before enrolling in a 300-level course.",
+                  "double_counting_allowed": true,
+                  "course_list": [
+                    "ITA 208",
+                    "ITA 209",
+                    "ITA 220",
+                    "ITA 225"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Departmentals",
+              "max_counted": 1,
+              "min_needed": 5,
+              "explanation": "All four tracks above require students to take eight upper-division courses. Five of these must be departmental courses, taken in the language of concentration.",
+              "req_list": [
+                {
+                  "name": "200-Level Courses",
+                  "max_counted": 2,
+                  "min_needed": null,
+                  "explanation": "Up to two of the five departmental courses may be taken at the 200-level (but FRE 207 and 208, or ITA 207 and 208, cannot both be counted).",
+                  "req_list": [
+                    {
+                      "name": null,
+                      "max_counted": null,
+                      "min_needed": null,
+                      "explanation": null,
+                      "course_list": [
+                        "ITA 2*",
+                      ],
+                      "excluded_course_list": [
+                        "ITA 207",
+                        "ITA 208"
+                      ]
+                    },
+                    {
+                      "name": null,
+                      "max_counted": 1,
+                      "min_needed": null,
+                      "explanation": null,
+                      "course_list": [
+                        "ITA 207",
+                        "ITA 208"
+                      ]
+                    }
+                  ]
+                },
+                {
+                  "name": "Upper-Level Courses",
+                  "max_counted": null,
+                  "min_needed": 3,
+                  "explanation": null,
+                  "course_list": [
+                    "ITA 3*",
+                    "ITA 4*",
+                    "ITA 5*"
+                  ]
+                }

```

Comment:
Not as pressing of an issue, but I noticed a small bug similar to the LANG bug where the "Find Satisfying Courses" returns 100-level courses even though this only encodes 300-level and higher.

Reply:
I pushed a fix for this [here](https://github.com/TigerPathApp/tigerpath/pull/324), so maybe you can try to see if the problem goes away if you rebase on master.